docs/source/posts/2022/2022-08-01-week-7-shivam.rst

```diff
@@ -0,0 +1,39 @@
+Week 7 - Fixing bugs in animations
+==================================
+
+.. post:: August 01 2022
+   :author: Shivam Anand
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+- This week I started with implementing scaling to the animation example.
+
+- I had a meeting with Mohamed in which we discussed the rotation issue, and we found out that glTF uses ``Slerp`` interpolator instead of the ``LinearInterpolator``. Using ``Slerp`` interpolator for rotation fixed the rotation issue.
+
+- Another issue we faced was with the multi-actor system. Some actors weren't rotating or translating as intended. This was caused because the transformations were applied to the ``polydata`` before creating an actor; Mohamed suggested applying transformation after the actor is created from polydata (keeping the center to origin).
+
+
+  .. raw:: html
+
+        <iframe id="player" type="text/html"   width="640" height="360" src="https://user-images.githubusercontent.com/74976752/182166133-585d06ef-55ff-48db-8ce7-98b377ebf8ec.mp4" frameborder="0"></iframe>
+
```

Comment:
here, add a link to the Kronos gif to show the comparison.

Reply:
Sure, I'll add this